README.md

```diff
@@ -0,0 +1,135 @@
+<a name="readme-top"></a>
+
+# Hello React Front End
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started ](#-getting-started-)
+  - [Setup](#setup)
+- [👥 Authors](#authors)
+- [🔭 Future Features ](#-future-features-)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖Hello React Back End <a name="about-project"></a>
+
+**Hello React Front End is a random greeting API. I used React for the front end and Ruby on Rails for back [hello-rails-back-end](https://github.com/batoorsohail/hello-rails-back-end)**
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>JavaScript</summary>
+  <ul>
+    <li><a href="#">React</a></li>
+  </ul>
+</details>
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Getting random greetings**
+- **Can use End Point API**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites <a name="prerequisites"></a>
+
+In order to run this project you need a Chrome, Safari or Mozilla browser
+
+To get a local copy up and running, follow these steps.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd <desired-folder>
+  git clone https://github.com/batoorsohail/hello-react-front-end.git
+```
+
+### Install <a name="install"></a>
+
+```sh
+  cd hello-react-front-end
+  bundle install
+```
+
+### Usage <a name="usage"></a>
+
+```sh
+  rails s
+```
+
+### Test <a name="test"></a>
+
+```sh
+  rspec spec
+```
+
```

Comment:
- [ ] @batoorsohail Great work on documenting both the front end and backend, however, you used the instructions of running the rails app on the front end instead of instructions for running a react app kindly consider changing it 
![image](https://github.com/batoorsohail/hello-rails-back-end/assets/61408860/a82a25fa-155f-4218-8cfa-68dce2592c92)